docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -314,7 +314,11 @@ image::sak-create-topic.png[Image of wizard to create a topic]
 
 After you complete the topic setup, the new Kafka topic is listed in the topics table. You can now start producing and consuming messages to and from this topic using services that you connect to this instance.
 
-NOTE: If your topic creation is unsuccessful and you receive an error message, try to create your topic again later. This situation might occur, for example, if your selected cloud provider has a temporary availability problem that affects your Kafka instance.
+ifndef::community[]
```

Comment:
In this case, almost all of the value in the note comes from the link to the Red Hat service limits article. Therefore, I've conditionalized it as non-community only.